Source/SubtitleProvider.h

```diff
@@ -25,7 +25,7 @@ class SubtitleProvider :
 public:
     TimedMetadataTrack SubtitleTrack = { nullptr };
 
-    winrt::weak_ref<MediaPlaybackItem> PlaybackItemWeak = { nullptr };
+    MediaPlaybackItem PlaybackItem = { nullptr };
```

Comment:
This change is not right. It is undoing part of the auto ref PR. We need weak ref here to avoid circle reference.